db/migrate/20221202115516_add_user_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Similarly, as per the given ERD, the reference name should be `author` instead of `user`, hence creating a migration query should be similar to:

```ruby
...
  def change
    add_reference :likes, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...